keras_nlp/models/deberta_v3/deberta_v3_tokenizer.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, proto, **kwargs):
         cls_token = "[CLS]"
         sep_token = "[SEP]"
         pad_token = "[PAD]"
+        mask_token = "[MASK]"
         for token in [cls_token, pad_token, sep_token]:
```

Comment:
I think we should add a check for the mask token here, which might also mean you need to update some unit tests for the preprocessor and tokenizer layers (so they add a mask token to the vocabulary).

Reply:
So, I added the check and as you can see it fails the preset tests, because apparently there isn't a `[MASK]` token in the `deberta_v3_extra_small_en` vocabulary, which seems strange to me (I've tried with `deberta_v3_base_en` too). Any advice?